### ISSUE TITLE
Action buttons were dark red after regaining consciousness despite being clickable 

### ISSUE THREAD

**Issue by cacogen** [tgstation/tgstation#49959]
## Round ID:

Round ID: 132239
BYOND Version: 513.1508
Server revision compiled on: 
Master commit: 6a599a213b447d2b063d9b7e3e4f98955e2a30b1

Local commit: 14e141a5991084ebd09afca63e5cf9ac4bdab13e
Server tools version: /tg/station 13 Server v3.2.6.2
## Testmerges:
The following pull requests are currently test merged:
#49344: '[READY] Space Dragon Rework' by IndieanaJones at commit 8416daa7c4

## Reproduction:

1. I had internals, a welding gas mask, and a PDA that had action buttons
2. I was put into crit
3. After regaining consciousness, every button was still dark red as though unavailable

Clicking the 'Toggle welding mask' button worked and changed the icon back to its normal available state.

**Comment by nemvar** [comment]
Has been a thing for ages and is likely a pain to fix.

**Comment by cacogen** [comment]
Yeah I didn't mean to imply it was anything new I'm just choosing to try to report bugs I encounter now instead of ignoring them like most people
